db/compaction/compaction_job.cc

```diff
@@ -1024,6 +1025,8 @@ void CompactionJob::ProcessKeyValueCompaction(SubcompactionState* sub_compact) {
       RecordDroppedKeys(range_del_out_stats,
                         &sub_compact->compaction_job_stats);
     }
+    // Status is checked in while loop.
```

Comment:
Since I agree that the status is checked in the while loop, I am wondering why these lines are necessary?

Reply:
I checked again, it's not needed so removed it.